doc/source/security/encrypting-threads.rst

```diff
@@ -0,0 +1,71 @@
+.. _encrypting-threads:
+
+================================================================================
+Working with Background Encryption Threads
+================================================================================
+
+:Availabiliity: This feature is **Experimental**.
+
+Encryption threads in the background allow you to perform some encryption and decryption tasks in real-time. 
+
+You would use encryption threads for the following purposes:
+
+* Encryption threads can encrypt existing tables in a tablespace. Encryption threads allow encryption to be applied to all or some of the existing tables, you can exclude tables from rotation, in a background process. You can encrypt existing tables with the Master key, but you must do this operation by tablespace.
+
+* Encryption threads encrypt tables with a key from a keyring. The Master key encrypts tables by a key and is stored in the encryption header of the tablespace.
+
+* Encryption threads allow key rotation. In an encryption thread rotation, the operation re-encrypts each tablespace page by page. The Master key rotation does not re-encrypt each page,  only the tablespace encryption header.
```

Comment:
only the tablespace encryption header - should be only tablespace key stored in tablespace encryption header.